Backend/app/dashboard/provider-dashboard/src/API/ProviderPlatform/DynamicOfferDriver/Driver/Registration.hs

```diff
@@ -100,3 +108,19 @@ registerRC merchantShortId apiTokenInfo driverId req =
     transaction <- buildTransaction Common.RegisterRCEndpoint apiTokenInfo driverId (Just req)
     T.withTransactionStoring transaction $
       Client.callDriverOfferBPP checkedMerchantId (.drivers.registerRC) driverId req
+
+generateAadhaarOtp :: ShortId DM.Merchant -> ApiTokenInfo -> Id Common.Driver -> Common.GenerateAadhaarOtpReq -> FlowHandler Common.GenerateAadhaarOtpRes
+generateAadhaarOtp merchantShortId apiTokenInfo driverId req =
+  withFlowHandlerAPI $ do
+    checkedMerchantId <- merchantAccessCheck merchantShortId apiTokenInfo.merchant.shortId
+    transaction <- buildTransaction Common.GenerateAadhaarOtpEndpoint apiTokenInfo driverId (Just req)
```

Comment:
I think this request will contain Aadhaar number, no need to save req. Pass Nothing instead of `Just req`